### ISSUE TITLE
How to cite this software

### ISSUE THREAD

**Issue by johnsgomez** [ccbogel/QualCoder#81]
Hello, mr. Curtain. I'm John S. Gómez from Colombia. I'm an undergraduate student of an English Language Teaching program, and I'm using QualCoder to analyze the data I have collected for my dissertation. I would like to know how can I give you credit for the software I have used for analyzing data. 

PS: I'm a beginner in Python and I like to take a look at the code to understand how it works. Thank you so much for this piece of software!

**Comment by ccbogel** [comment]
Hello John

Thank you for using QualCoder.

How did you find QualCoder for your analyses, I would like to gain feedback?

I have created a new release which was needed anyways, so you could reference this release by including something like this in your citation, which follows the APA style:

Curtain, C. (2020) QualCoder 1.9 [Computer software]. Retrieved from
https://github.com/ccbogel/QualCoder/releases/tag/1.9

with regards Colin

p.s. I added my university contact details below - in case it was needed


Dr Colin Curtain BPharm(Hons) GradDipComp PhD MPS

Lecturer, MClinPharm course coordinator

School of Pharmacy & Pharmacology |College of Health and Medicine

University of Tasmania

Private Bag 26, Hobart TAS 7001

Tel: +61 3 6226 1096| Fax: +61 3 6226 7627 Email: Colin.Curtain@utas.edu.au<mailto:colin.curtain@utas.edu.au>


________________________________

**Comment by johnsgomez** [comment]
Hello, thanks for replying. 

Before using QualCoder I had previously used ATLAS.TI and was familiar with RQDA. I liked the fact that RQDA was an open source software, but the interface was not very intuitive and the functionality was kind of limited. ATLAS.TI was more suitable for me but it has the problem of needing a license, which is expensive and not always available at my university. Additionally, ATLAS.TI doesn't allow you to export the complete database of documents, quotations and cites. I started to look for open source projects and I found QualCoder. I tried to install it on a Windows OS, but I was not able to do it, but at the end I could made it work on a Linux OS.

What I like the most about QualCoder is that it allows you to work directly with your data base, this allows you to make complex operations. I'm not very familiar with SQL, but I have exported my database to a csv file and worked with it using Pandas. By doing this, I have been able to look for co-occurrence of codes, and this has been very helpful in my research project. 

I think that somethings could be improved in QualCoder, to make coding more intuitive and fast, and for you to be able to include a larger amount of codes. However, in general I have been able to analyze data in a way that I was not able using any other software. 

Thank you, I hope you find this piece of feedback useful. I would be happy to provide more feedback later on, and contribute to this project.
